app/dashboard/bookmarks/page.tsx

```
@@ -0,0 +1,47 @@
+import { currentUser } from '@clerk/nextjs/server';
+import { redirect } from 'next/navigation';
+import { Button } from '@/components/ui/button';
+import { ChevronLeft } from 'lucide-react';
+import Link from 'next/link';
+import { BookmarkedProviders } from '@/components/bookmarks/bookmarked-providers';
```

ESLint

[error] 'BookmarkedProviders' is defined but never used.

(@typescript-eslint/no-unused-vars)